taskvine/src/tools/vine_plot_txn_log

```diff
@@ -0,0 +1,467 @@
+#! /usr/bin/env python
+
+# Copyright (C) 2022- The University of Notre Dame
+# This software is distributed under the GNU General Public License.
+# See the file COPYING for details.
+
+from collections import Counter
+import matplotlib.pyplot as plt
+import time
+import sys
+import argparse
+import json
+
```

Comment:
Let's add this style for accessibility:

`plt.style.use('tableau-colorblind10')`